ceremonies/src/tests.rs

```diff
@@ -1253,6 +1253,49 @@ fn get_assignment_params_works() {
 	});
 }
 
+#[test]
+fn get_inactive_communities_works() {
+	new_test_ext().execute_with(|| {
+		let cid0 = CommunityIdentifier::default();
+		let cid1 = CommunityIdentifier::new(
+			Location::new(Degree::from_num(1f64), Degree::from_num(1f64)),
+			Vec::<i64>::new(),
+		)
+		.unwrap();
+
+		let mut cindex = 5;
+		let assignment =
+			AssignmentCount { bootstrappers: 1, reputables: 0, endorsees: 0, newbies: 0 };
+		AssignmentCounts::insert((cid0, cindex), assignment);
```

Comment:
#107 demands successful meetups. You are only checking assignment of meetups which might not actually have taken place. I guess you should check Reputation instead oft assignments. Or introduce a flag to signal at least one successful meetup has been verified and issued

Reply:
how about checking attestationcount!=0 of the last cindex during registration phase?

Reply:
No. Attestations must be validated. Can't be trusted

Reply:
Check Reputation registry or introduce a flag if that is too expensive